docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -6,7 +6,7 @@ beta[]
 
 --
 
-Alerting allows you to detect complex conditions within different {kib} apps and trigger actions when those conditions are met. Alerting is integrated with <<xpack-apm,*APM*>>, <<metrics-app,*Metrics*>>, <<xpack-siem,*Security*>>, <<uptime-app,*Uptime*>>, can be centrally managed from the <<management,*Management*>> UI, and provides a set of built-in <<action-types, actions>> and <<alert-types, alerts>> for you to use.
+Alerting allows you to detect complex conditions within different {kib} apps and trigger actions when those conditions are met. Alerting is integrated with <<xpack-apm,*APM*>>, <<metrics-app,*Metrics*>>, <<xpack-siem,*Security*>>, <<uptime-app,*Uptime*>>, can be centrally managed from the <<management,*Management*>> UI, and provides a set of built-in <<action-types, actions>> and <<alert-types, alerts>> (known as *Stack Alerts*) for you to use.
```

Comment:
I don't think "(known as stack alerts)" is needed here.  The link takes you to a page that explains the different types of alerts.

We should add the terms "stack alert" and "domain-specific alert" to the [glossary.](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html)

Reply:
I want to hammer on the terminology here since it is related to RBAC.  Putting it in the glossary kind of buries it.